src/config/gen.ml

```diff
@@ -0,0 +1,22 @@
+(**********************************************************************
```

Comment:
Shouldn't this be in `tools/gen_compat.ml` ? Near `gen_version.ml`.

Reply:
The other one is in ocamlformat_support, which is kind of an other project. I understand the reason (local) but it feels weird to have it in the `src` directory.

Reply:
You've convinced me.